config/locales/eng/models.yml

```diff
@@ -190,6 +190,8 @@ eng:
           unvalid_amounts: "The journal entry must be filled either debit or credit but not at the same time."
         outgoing_parcel_item:
           greater_than_undelivered_quantity: "The quantity of %{product} to deliver is greater than the available quantity (%{maximum}%{unit})"
+        purchase:
+          payment_delay_custom_validation_message: "Correct formats : eom, end of month, %{values}"
```

Comment:
`eom` and `end of month` should be in the interpolation too.

(With something like `values: TRANSLATIONS[...].... + ["eom", "end of month"]` probably)

Also, `bom`, `beginning of month` and `ago` should be in the list too.